src/Common/Core/Impl/Telemetry/TelemetryServiceBase.cs

```diff
@@ -32,13 +32,13 @@ public abstract class TelemetryServiceBase : ITelemetryService, IDisposable {
         /// </summary>
         public bool IsEnabled {
             get {
-                return this.TelemetryRecorder.IsEnabled;
+                return this.TelemetryRecorder?.IsEnabled == true;
```

Comment:
There are `TestTelemetryRecorder` and `FileTelemetryRecorder` that should be created for the tests. Do you remember what tests were failing?

---
In reply to: [74832126](https://github.com/Microsoft/RTVS/pull/2173#discussion_r74832126) [](ancestors = 74832126,74828037)